ci_framework/roles/edpm_prepare/tasks/main.yml

```
@@ -105,6 +105,23 @@
           --namespace={{ cifmw_install_yamls_defaults['OPERATOR_NAMESPACE'] }}
           --for=jsonpath='{.status.phase}'=Complete --timeout=20m
 
+    # TODO(pablintino): rework that part in order to use an Assert
+    # this means mostly: push this to a new task file, and retry to include_tasks
+    # The new task file may use some block/rescue in order to properly display the failure if any
+    - name: Make sure all Openstack operators are deployed
+      environment:
+        KUBECONFIG: "{{ cifmw_openshift_kubeconfig }}"
+        PATH: "{{ cifmw_path }}"
+      ansible.builtin.shell: |
```

```suggestion
      ansible.builtin.shell: >-
```
That would avoid the need to use `\`